rampwf/utils/tests/test_testing.py

```diff
@@ -0,0 +1,22 @@
+import shutil
+from tempfile import mkdtemp
+from rampwf.kits import fetch_ramp_kit
+from rampwf.utils import assert_submission
+
+# later on we need to make a util to list kits to avoid repeating the same
+# list everywhere
+RAMP_KITS = ('boston_housing',
```

Comment:
As my comment said, I am unsure about this part :D. The only issue with your solution is that we might not want to test all the kits present in `ramp_kits.py`? What other solutions could we have here?